step9_stop.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
You are trying to solve problems you've just created.

Reply:
I can run apps in docker? yes

Reply:
So you can run use microservices, why not to use them here?